### ISSUE TITLE
v2023.12.0-alpha-2 java error

### ISSUE THREAD

**Issue by jospoortvliet** [FAForever/downlords-faf-client#3079]
App takes a while to try and launch, then...

```
java.lang.RuntimeException: Exception in Application start method
	at com.sun.javafx.application.LauncherImpl.launchApplication1(LauncherImpl.java:893)
	at com.sun.javafx.application.LauncherImpl.lambda$launchApplication$2(LauncherImpl.java:196)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.lang.IllegalArgumentException: Invalid URL: Invalid URL or resource not found
	at javafx.scene.image.Image.validateUrl(Image.java:1133)
	at javafx.scene.image.Image.<init>(Image.java:627)
	at com.faforever.client.fx2j.theme.PersistentNotificationBuilder.build(PersistentNotificationBuilder.java:138)
	at com.faforever.client.fx2j.theme.PersistentNotificationBuilder.build(PersistentNotificationBuilder.java:28)
	at io.github.sheikah45.fx2j.api.Fx2jLoader.load(Fx2jLoader.java:146)
	at com.faforever.client.theme.UiService.loadFxml(UiService.java:73)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:352)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:713)
	at com.faforever.client.theme.UiService$$SpringCGLIB$$0.loadFxml(<generated>)
	at com.faforever.client.notification.PersistentNotificationsController.addNotification(PersistentNotificationsController.java:50)
	at java.base/java.lang.Iterable.forEach(Iterable.java:75)
	at java.base/java.util.Collections$UnmodifiableCollection.forEach(Collections.java:1116)
	at com.faforever.client.notification.PersistentNotificationsController.onInitialize(PersistentNotificationsController.java:46)
	at com.faforever.client.fx.Controller.initialize(Controller.java:27)
	at com.faforever.client.fx2j.theme.PersistentNotificationsBuilder.build(PersistentNotificationsBuilder.java:69)
	at com.faforever.client.fx2j.theme.headerbar.NotificationButtonBuilder.build(NotificationButtonBuilder.java:66)
	at com.faforever.client.fx2j.theme.headerbar.HeaderBarBuilder.build(HeaderBarBuilder.java:215)
	at com.faforever.client.fx2j.theme.MainBuilder.build(MainBuilder.java:64)
	at com.faforever.client.fx2j.theme.MainBuilder.build(MainBuilder.java:21)
	at io.github.sheikah45.fx2j.api.Fx2jLoader.load(Fx2jLoader.java:146)
	at com.faforever.client.theme.UiService.loadFxml(UiService.java:73)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:352)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:713)
	at com.faforever.client.theme.UiService$$SpringCGLIB$$0.loadFxml(<generated>)
	at com.faforever.client.FafClientApplication.showMainWindow(FafClientApplication.java:87)
	at com.faforever.client.FafClientApplication.start(FafClientApplication.java:72)
	at com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$9(LauncherImpl.java:839)
	at com.sun.javafx.application.PlatformImpl.lambda$runAndWait$12(PlatformImpl.java:483)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:456)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:455)
	at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
	at com.sun.glass.ui.gtk.GtkApplication._runLoop(Native Method)
	at com.sun.glass.ui.gtk.GtkApplication.lambda$runLoop$10(GtkApplication.java:263)
	... 1 more
Caused by: java.lang.IllegalArgumentException: Invalid URL or resource not found
	at javafx.scene.image.Image.validateUrl(Image.java:1122)
	... 38 more
```

Not sure if it gives any helpful info for you to fix, but I thought I'd share. The last stable version works just fine, so I'm good with that for now of course.

**Comment by jospoortvliet** [comment]
Thanks @Sheikah45

**Comment by Sheikah45** [comment]
The fix is released with the 2023.12.0-alpha-3
